charmhelpers/fetch/ubuntu.py

```diff
@@ -574,6 +622,10 @@ def add_source(source, key=None, fail_invalid=False):
       with be used.  If staging is NOT used then the cloud archive [3] will be
       added, and the 'ubuntu-cloud-keyring' package will be added for the
       current distro.
+    '<openstack-version>': translate to cloud:<release> based on the current
+      distro version (i.e. for 'ussuri' this will either be 'bionic-ussuri' or
+      'distro'.
+    '<openstack-version/proposed': as above, but for proposed.
```

Comment:
Should this say: `'<openstack-version>/proposed'`?

Reply:
> Should this say: `'<openstack-version>/proposed'`?

Ack, yes, it should.  Good catch, thank you.